src/index.html

```diff
@@ -6,7 +7,42 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+  <!--endregion-->
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-fri calendar--month-length-29">
+
+      <div class="calendar-day"></div>
```

Comment:
```suggestion
    <div class="calendar calendar--start-day-fri calendar--month-length-29">
      <div class="calendar-day"></div>
```

Redundant empty line between parent and child elements